app/src/main/kotlin/com/softteco/template/ui/feature/login/LoginScreen.kt

```diff
@@ -124,6 +125,14 @@ private fun ScreenContent(
                     },
                     style = MaterialTheme.typography.bodyLarge
                 )
+                ClickableText(
+                    text = AnnotatedString(stringResource(id = R.string.forgot_password)),
+                    modifier = Modifier.padding(Dimens.PaddingDefault),
```

Comment:
It's better to avoid non-obvious padding declarations. The first ClickableText has padding around it. The second one does as well. In fact, the padding between the two widgets consists of two values. It is more obvious to use spacer or declare padding between widgets for only one of them. For example:
```
fun Column{
    Text()
    Text(padding(top = PaddingDefault))
    Text(padding(top = PaddingDefault))
}
```
or

```
fun Row{
    Text()
    Text(padding(start = PaddingDefault))
    Text(padding(start = PaddingDefault))
}
```

Reply:
fixed